modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json

```diff
@@ -277,7 +283,7 @@
 																			"value": {
 																				"html": {
 																					"mapping": {
-																						"fieldKey": "ddmTemplate__ddmTemplate_8811548",
+																						"fieldKey": "AssetVocabulary_47184",
```

Comment:
I think this needs to be replaced. Please, check with Nilton about which replacement you can use here.